### PR TITLE
Adding example options for variable

### PR DESCRIPTION
**Pull request by jhoyt-rh**
@xltian Neither Anping nor Gilbert are accepted as reviewers in Bugzilla, so I had to send this to you, QA contact. I will also email Gilbert and Anping about this PR.

This is for https://bugzilla.redhat.com/show_bug.cgi?id=1634097

@vikram-redhat PTAL
@openshift/team-documentation PTAL

This BZ was found in and reported against Version 3.10, but applies to 3.11 as well.

### CODE REVIEW

**Comment by jhoyt-rh** [comment]
@Gilbert Kimetto , I forgot to include you in the review. PTAL. Thank you.

File: install/configuring_inventory_file.adoc [hunk 1]
```diff
@@ -455,12 +455,13 @@ Groups and Host Mappings] for the current method of setting node labels.
 a|This variable configures additional `docker` options in
 *_/etc/sysconfig/docker_*, such as options used in
 xref:host_preparation.adoc#managing-docker-container-logs[Managing Container Logs].
-It is recommended to use `json-file`.
+We recommend that you use `json-file`.
 
 The following example shows the configuration of Docker to use the `json-file` log
-driver, where Docker will rotate between three 1 MB log files:
+driver, where Docker can rotate between three 1 MB log files, and signature verification is not required.
+When supplying additional options, be sure you maintain the single quotation mark formatting:
```
Comment (mburke5678) [inline install/configuring_inventory_file.adoc]: I think we use _ensure_ over _be sure_ by a margin 1780-to-5 based on an Atom search. 
_ensure you maintain_ ??

File: install/configuring_inventory_file.adoc [hunk 2]
```diff
@@ -455,12 +455,13 @@ Groups and Host Mappings] for the current method of setting node labels.
 a|This variable configures additional `docker` options in
 *_/etc/sysconfig/docker_*, such as options used in
 xref:host_preparation.adoc#managing-docker-container-logs[Managing Container Logs].
-It is recommended to use `json-file`.
+We recommend that you use `json-file`.
```
Comment (kalexand-rh) [inline install/configuring_inventory_file.adoc]: The style guide says not to use "Do not use to refer to a company (for example, “We recommend...”)."

It would be better to explain why you need to use `json-file`, otherwise revert the change.

File: install/configuring_inventory_file.adoc [hunk 3]
```diff
@@ -455,12 +455,13 @@ Groups and Host Mappings] for the current method of setting node labels.
...
 
 The following example shows the configuration of Docker to use the `json-file` log
-driver, where Docker will rotate between three 1 MB log files:
+driver, where Docker can rotate between three 1 MB log files, and signature verification is not required.
```
Comment (kalexand-rh) [inline install/configuring_inventory_file.adoc]: s/can rotate/rotates
s/files,/files

**Comment by mburke5678** [comment]
@jhoyt-rh One nit. Otherwise LGTM.

File: install/configuring_inventory_file.adoc (hunk 1 above)
Comment (kalexand-rh) [reply install/configuring_inventory_file.adoc]: I'd s/be sure you/ensure that you

File: install/configuring_inventory_file.adoc (hunk 2 above)
Comment (mburke5678) [reply install/configuring_inventory_file.adoc]: @kalexand-rh This is yet another issue we should get before the team. There are several constructs around recommended/not recommended looking in Atom.

File: install/configuring_inventory_file.adoc (hunk 1 above)
Comment (jhoyt-rh) [reply install/configuring_inventory_file.adoc]: HAHAHA!! OK, I can live with that! :-)

File: install/configuring_inventory_file.adoc (hunk 2 above)
Comment (jhoyt-rh) [reply install/configuring_inventory_file.adoc]: It is also something we are talking about in the Doc Conventions initiative -- looking at whether we should change that. I'll be sure to bring it up in the next meeting again. Thanks!

**Comment by jhoyt-rh** [comment]
@vikram-redhat 
@gkimetto@redhat.com
@anpingli 

Version question, need information, please:

Should this roll back to enterprise-3.9? I'm told the codeblock with --log_driver json_file started in 3.6?
IF SO, can I CP back to those, or do I have to create new PRs based on those releases? (This PR was based on master.)

Thanks!

**Review by anpingli — APPROVED** [review]

**Comment by anpingli** [comment]
@jhoyt-rh  I think it Ok to Backport  the  V3.9.

File: install/configuring_inventory_file.adoc [hunk 4]
```diff
@@ -455,12 +455,13 @@ Groups and Host Mappings] for the current method of setting node labels.
...
+When supplying additional options, be sure you maintain the single quotation mark formatting:
 ----
-"--log-driver json-file --log-opt max-size=1M --log-opt max-file=3"
+OPTIONS=' --selinux-enabled --log-opt  max-size=1M --log-opt max-file=3 --insecure-registry 172.30.0.0/16 --log-driver=json-file --signature-verification=False'
```
Comment (vikram-redhat) [inline install/configuring_inventory_file.adoc]: @anpingli - does it matter whether the variable is "False" instead of "false"?

**Comment by anpingli** [comment]
Either of them work.  It is 'false' by default.

**Comment by kimettog** [comment]
Looks good to me

File: install/configuring_inventory_file.adoc (hunk 1 above)
Comment (jhoyt-rh) [reply install/configuring_inventory_file.adoc]: DONE

File: install/configuring_inventory_file.adoc (hunk 4 above)
Comment (jhoyt-rh) [reply install/configuring_inventory_file.adoc]: Either is OK, but I changed to the default, which is lowercase.

**Comment by jhoyt-rh** [comment]
added enterprise-3.9 label, based on QE answer

**Comment by jhoyt-rh** [comment]
Going to create a new PR just for enterprise-3.9
